packages/messaging/test/index.test.ts

```diff
@@ -18,9 +18,12 @@ import { expect } from 'chai';
 import { sandbox, SinonSandbox, SinonStub } from 'sinon';
 
 import { FirebaseApp } from '@firebase/app-types';
-import { _FirebaseNamespace } from '@firebase/app-types/private';
+import {
+  _FirebaseNamespace,
+  FirebaseServiceFactory
+} from '@firebase/app-types/private';
```

Comment:
It'd be really nice to have a test that works like [the docs](https://github.com/firebase/firebase-js-sdk/blob/master/packages/messaging/README.md) suggests a user to, i.e. 

```
import firebase from '@firebase/app';
import '@firebase/messaging'
// Do stuff w/ `firebase` and `firebase.messaging`
```
rather than poking at the internals like this :/

Reply:
Yeah, I agree. The tests need an overhaul (not just here). I'll get to that soon™.